app/(candidate)/dashboard/polls/components/PollImageUpload.tsx

```diff
@@ -0,0 +1,136 @@
+'use client'
+import { useRef, useState } from 'react'
+import Image from 'next/image'
+import { LuCloudUpload } from 'react-icons/lu'
+import { CircularProgress } from '@mui/material'
+import { HiddenFileUploadInput } from '@shared/inputs/HiddenFileUploadInput'
+import { uploadFileToS3 } from '@shared/utils/s3Upload'
+import { apiRoutes } from 'gpApi/routes'
+
+const FILE_LIMIT_MB = 5
+const ACCEPTED_FORMATS = ['.png', '.jpg', '.jpeg']
+
+export const PollImageUpload: React.FC<{
```

Comment:
Since this is being used in two different places, that would suggest that it's a genericized component and should probably be named something more agnostic like `ImageUpload` and would probably be better organized if it were in the `app/shared` folder as opposed to being specific to the Polls feature